Model/Sync/Catalog/Processor.php

```diff
@@ -715,8 +715,8 @@ public function retryProductSync()
      * Check and sync the products if not already synced
      *
      * @param array <mixed> $productIds
-     * @param array $visibleItems
-     * @param string $storeId
+     * @param array <mixed> $visibleItems
+     * @param int|null|string $storeId
```

Comment:
Hi @manaftryzens can you please explain why this argument can be of type `int|null|string`?

Reply:
As per magento's core code which returns the storeId from an order it's documented as it  @return int|null . So to accommodate with the coding standard, we should add that type in the related functions too.

Reference : Magento\Sales\Model\Order::getStoreId() method

Reply:
Okay, so what is the use case of it being `string`?

Reply:
Magento\Store\Model\Store::getId() calls Magento\Framework\DataObject::_getData($key), which returns mixed

Reply:
HI @EranAvidor ,

Sorry, Actually there isn't a case of string in this code. We are passing storeId from either quote object or order object. It is already converting to (int) at Magento\Quote\Model\Quote::getStoreId() method. I will send a new PR correcting this.

Reply:
Please approve this new PR below

https://github.com/YotpoLtd/magento2-module-core/pull/112

Reply:
Approved and merged.
Thanks @manaftryzens.